### PR TITLE
Add SSH issue and sign operations to SDK

### PR DESCRIPTION
**Pull request by dangtony98**
Added upcoming SSH `/issue` and `/sign` endpoints to the Go SDK.

This is useful for applications that need to obtain SSH credentials.

### CODE REVIEW

File: packages/models/ssh.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+	HostCert SshCertType = "host"
+)
+
+// type SignedSshKeyWithKeyPair struct {
+// 	SerialNumber  string           `json:"serialNumber"`  // Unique serial number for the SSH key
+// 	SignedKey     string           `json:"signedKey"`     // Signed SSH public key (certificate)
+// 	PrivateKey    string           `json:"privateKey"`    // Corresponding private key
+// 	PublicKey     string           `json:"publicKey"`     // Corresponding public key
+// 	KeyAlgorithm  CertKeyAlgorithm `json:"keyAlgorithm"`  // Algorithm used for the key
+// }
```
Comment (sheensantoscapadngan) [inline packages/models/ssh.go:25]: do we still need this?

File: ssh.go [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+func (f *Ssh) SignSshPublicKey(options SignSshPublicKeyOptions) (string, error) {
+	res, err := api.CallSignSshPublicKeyV1(f.client.httpClient, options)
+
+	if err != nil {
+		return "", err
+	}
+
+	return res.SignedKey, nil
+}
```
Comment (sheensantoscapadngan) [inline ssh.go:27]: just for flexibility, can we also return the serial number? basically just the entire `SignPublicKeyResponse` schema

File: packages/models/ssh.go (hunk 1 above)
Comment (dangtony98) [reply packages/models/ssh.go:25]: Removed

File: ssh.go (hunk 2 above)
Comment (dangtony98) [reply ssh.go]: Updated

**Review by sheensantoscapadngan — APPROVED** [review]

**Pull request merged**